style.css

```diff
@@ -8,6 +8,80 @@
   box-sizing: border-box;
 }
 
+.modal-buttons {
+  padding: 2rem;
+  width: 100%;
+  display: flex;
+  flex: 1;
+  justify-content: space-between;
+}
+
+.modal-buttons a {
+  text-decoration: none;
+  color: white;
+}
+
+.modal-buttons button {
+  width: 48%;
+}
+
```

Comment:
- [ ] `required`
- [ ] Amazing job with the pop up so far :clap: :clap: 
- [ ] How ever I suggest you stick to the figma designer. why? because `Clients or company should come first in everything` in future you will to Organize each project around your clients’/company needs, goals, strategies and expectations. We emphasize communication, as it is critical for the success of any project. Whatever your client’s/company pain points are, address them and provide viable solutions. such as design specifications e.t.c 
- Kindly adjust the `buttons` `width` `background blur` and social `icons` on the `buttons`
- please see screenshots below 

`Your image`
![Screenshot from 2022-07-20 18-41-19](https://user-images.githubusercontent.com/66230306/180040647-6dfa0665-006c-45ab-ac44-8249ba2f304b.png)

`figma image`
![Screenshot from 2022-07-20 18-42-15](https://user-images.githubusercontent.com/66230306/180040943-58a0432b-57ef-4509-bfd8-19ba938acc90.png)